runtime/src/qio/qio_formatted.c

```diff
@@ -417,14 +484,14 @@ qioerr qio_channel_read_string(const int threadsafe, const int byteorder, const
     case QIO_BINARY_STRING_STYLE_LENvB_DATA:
       err = qio_channel_read_uvarint(false, ch, &num);
       break;
-    case QIO_BINARY_STRING_STYLE_TOEOF:
-      // read until the end of the file.
-      // Figure out how many bytes are available.
-      err = _peek_until_len(ch, maxlen, &peek_amt);
-      num = peek_amt;
-      // Ignore EOF errors as long as we read something.
-      if( err && qio_err_to_int(err) == EEOF && num > 0 ) err = 0;
-      break;
+//     case QIO_BINARY_STRING_STYLE_TOEOF:
```

Comment:
Probably best to leave the case here but have it assert and break?

Reply:
What should I assert for ?

Reply:
I apologise Sir but if you can kindly explain the point a bit more ? I mean what is it that I should be checking in assert ? The assertion will be that this case has already been handled ?? but then , in that case,it won't be entering the if statement because I am returning from that if block itself .

Reply:
Yes, the case should be handled above. If it is not, there is a bug, and that will be easier to fix if there is an assert. Think of it as a way to help future developers making changes in this area.

Reply:
Ok, I shall make this change once you are able to see the other changes I have made .

Reply:
Still need to do something here

Reply:
@mppf  I have added a break statement . Shouldnt that suffice ?

Reply:
it is fine. next step is to merge all of your commits into one and make a PR against the main Chapel repository.

Reply:
@mppf  Ok..Kindly allow me some days for this.